### PR TITLE
ENH: migrate to HUGO!

### PR DESCRIPTION
**Pull request by lwasser**
getting things ready so we can add the pyos packaging lessons here!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,170 @@
-# dsk
-lesson website
+# Welcome to the datascienceskills.org repository!
```
Comment (lwasser) [inline README.md:1]: Missing environment instructions now

File: .github/workflows/build-site.yml [hunk 2]
```diff
@@ -14,23 +14,44 @@ jobs:
...
-          quarto render dsk/
           ls
 
-      # Push the book's HTML to github-pages
+      - name: Build quarto/hugo website
+        continue-on-error: false
+        run: |
+          # Render quarto qmd files --> md
+          quarto render
+          npm run build
```
Comment (lwasser) [inline .github/workflows/build-site.yml:48]:
```suggestion
        run: |
          # Render quarto qmd files --> md
          #quarto render
          npm run build
```

File: content/english/lessons/git-github/git-intro/2-what-is-version-control.qmd [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+estimated-time: "2 3 hours"
+difficulty: "beginner"
+module: "intro-git"
+url: /git-intro/about-version-control.html
+order: 2
+---
+
+```{=markdown}
+{{< toc >}}
+```
```
Comment (cderv) [inline content/english/lessons/git-github/git-intro/2-what-is-version-control.qmd]: Not ideal but for now, tirple brackets should be used to escape

```suggestion
{{{< toc >}}}
```

File: content/english/lessons/git-github/git-intro/2-what-is-version-control.qmd [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+- Implement version control using **git**.
+
+## <i class="fa-regular fa-square-check"></i> What You Need
+
+Be sure that you have completed the instructions on [Setting up Git, Bash, and Conda on your computer](/git/setup-git-bash) to install the tools for your operating system (Windows, Mac, Linux).
+
+You will also need a web browser and your **GitHub.com** login (username and password).
+
+{{{< /noticeowl >}}}
+```
```
Comment (cderv) [inline content/english/lessons/git-github/git-intro/2-what-is-version-control.qmd]:
```suggestion
{{{< noticeowl "learn" >}}}

After completing this module, you will be able to:

- Define **version control**.
- Explain why **version control** is useful in a scientific workflow.
- Implement version control using **git**.

## <i class="fa-regular fa-square-check"></i> What You Need

Be sure that you have completed the instructions on [Setting up Git, Bash, and Conda on your computer](/git/setup-git-bash) to install the tools for your operating system (Windows, Mac, Linux).

You will also need a web browser and your **GitHub.com** login (username and password).

{{{< /noticeowl >}}}
```

**Review by cderv — COMMENTED** [review]
For now you should use triple brackets syntax to escape shortcodes for Hugo to process

**Comment by cderv** [comment]
With next quarto 1.4 pre-release, the shortcodes unknown to Quarto should be left asis. So you would not need to escape them and it should work. 

Hope it helps with this shortcodes issue.

**Comment by lwasser** [comment]
pre-commit autofix

**Pull request merged**